husky_description/meshes/accessories/bumblebee2.dae

```diff
@@ -0,0 +1,297 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
removed the asset https://github.com/husky/husky/pull/33/commits/6ebc355d5ebda7a260aa809f494881b88d1e156e